.pre-commit-config.yaml

```diff
@@ -50,4 +50,5 @@ repos:
     name: PyLint
     files: \.py$
     entry: python3 -m pylint.__main__
-    args: []
+    args:
+    - --good-names=pk,urlpatterns
```

Comment:
Why `pk`? Where is it used?

Reply:
No, I will not use it anywhere. Sorry i will correct it.